packages/flutter_test/lib/src/window.dart

```diff
@@ -947,6 +973,122 @@ class TestFlutterView implements FlutterView {
   }
 }
 
+/// A version of [Display] that can be modified to allow for testing various
+/// use cases.
+class TestDisplay implements Display {
+  /// Creates a new [TestDisplay] backed by the given [Display].
+  ///
+  /// Updates to the [TestDisplay] will be surfaced through
+  /// [PlatformDispatcher.onMetricsChanged].
```

Comment:
This is probably more relevant on the class documentation instead of the constructor.

Reply:
Moved!